### ISSUE TITLE
File encoding

### ISSUE THREAD

**Issue by moi90** [ecotaxa/pyecotaxa#3]
It seems that the encoding of the TSV files generated is not necessarily UTF-8. My opinion is that this should be changed. However, which encoding is used?

**Comment by jiho** [comment]
Latin-1

**Comment by jiho** [comment]
But see https://github.com/oceanomics/ecotaxa_dev/issues/256

**Comment by moi90** [comment]
So what is the safest way to proceed for now? Read and write as ASCII by default so the responsibility lies with the user? Unfortunately, Latin-1 is not a subset of UTF-8...

**Comment by jiho** [comment]
I know there are ways to snif encoding based on file content but don't know much more.

I just enforce Latin-1 for now and will switch to UTF-8 when we finally make the switch. And it will be incompatible with old files which will force people to upgrade, see the light and use unicode; and that's fine by me.

**Comment by moi90** [comment]
Yes, maybe I can also include my encoding detection logic here that I use in morphocut. I don't want to include it directly into read_tsv because I don't want to lose the ability of `pandas.read_cvs` to directly access compressed files, URLs, etc.
